src/main/java/com/librato/metrics/client/CounterMeasure.java

```diff
@@ -15,7 +15,7 @@ public CounterMeasure(String name, long value) {
     public Map<String, Object> toMap() {
         Map<String, Object> map = super.toMap();
         Maps.putIfNotNull(map, "measure_time", epoch);
-        Maps.putIfNotNull(map, "source", Sanitizer.LAST_PASS.apply(source));
+        Maps.putIfNotNull(map, "source", Sanitizer.VALUE_SANITIZER.apply(source));
```

Comment:
the valid values for source aren't the same as the valid tag values (maybe they should be?)

See https://github.com/librato/jackdaw/blob/master/src/main/java/com/librato/jackdaw/validators/SourceValidator.java#L8